uptane-standard.md

```diff
@@ -868,32 +873,37 @@ In order to perform partial verification, an ECU SHALL perform the following ste
-    1. Locate and download a Targets metadata file from the Image repository that contains an image with exactly the same file name listed in the Director metadata, following the procedure in {{resolve_delegations}}.
-    2. Check that the Targets metadata from the Image repository matches the Targets metadata from the Director repository:
-        1. Check that the non-custom metadata (i.e., length and hashes) of the unencrypted image are the same in both sets of metadata.
-        2. Check that all "MUST match" custom metadata (e.g., hardware identifier and release counter) are the same in both sets of metadata.
-        3. Check that the release counter in the previous targets metadata file is less than or equal to the release counter in this targets metadata file.
+1. Load the repository mapping metadata ({{repo_mapping_meta}}).
+2. Load the latest verified time from the time server, if implemented.
+3. Refresh the top-level metadata for the Director repository, by following the steps in {#refresh_toplevel_metadata}, for the first repository in the list of repositories in the mapping metadata.
+4. If this ECU is a primary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** all image metadata.  The primary ECU SHOULD also ensure that the image metadata from the director repository doesn't contain any ECU identifiers for ECUs not actually present in the vehicle.
+5. If instead this ECU is a secondary ECU, then iterate through the image metadata listed in the Director repository's top-level Targets metadata, and **store** the first image metadata listed that includes this secondary ECU's ECU identifier and hardware ID.
+6. For each image name for which the previous two steps **stored** image metadata, obtain verified image metadata for that image by iterating over each mapping in the repository mapping metadata, in order, and performing the following steps for each mapping:
+    1. If the image name does not match the image paths listed in the mapping, skip to the next mapping.
+    2. For each repository listed in this mapping, if top-level metadata has not been refreshed for that repository using {#refresh_toplevel_metadata} in this iteration of the {#full_verification} procedure, refresh top-level metadata for that repository using the steps in {#refresh_toplevel_metadata}.
+    3. For each repository listed in this mapping, perform the procedure in {{resolve_delegations}}, beginning at the top-level Targets metadata in that repository, to find metadata for the image if it exists, and **store** it if it does.
+    4. Find the first set of at least the mapping's threshold number of repositories, including the Director repository, for which all of the following is true for the stored image metadata for each repository:
```

Comment:
I find this very hard to parse on first read: "Find the first set of at least the mapping's threshold number of repositories"

Reply:
Yeah, that might be a hard parse....
If the matching conditions didn't immediately follow, I might suggest "of at least X" followed later by "where X is...".   I'll try something else.

Reply:
Is this any better?

```
4. Find the first set of at least X repositories -- where X is the mapping's
threshold, and such that the Director repository is in the set -- for which all
of the following is true for the stored image metadata for each repository:
```

(While I was at it, I realized that the "including the Director repository" bit could be misinterpreted, so I clarified that, too.)  It still reads a bit complex..... Hm.